src/server/main.c

```
@@ -875,30 +922,43 @@ int main(int argc, char **argv)
+                    int mixed_rmt_len = -1;
+                    if (hlen_rmt > 0) {
+                        mixed_rmt_len = build_mixed_stateupdate_remote(
+                            &p->ship, g_game_time,
+                            hbuf_rmt, hlen_rmt,
+                            mixed_rmt, sizeof(mixed_rmt));
+                    }
+
+                    /* Advance cursor using the owner version (smaller budget,
+                     * may cover fewer entries -- that's fine, the remote
+                     * version just sends more data this tick). */
+                    if (hlen_own > 0)
```

Clang (14.0.6)

[note] +5, including nesting penalty of 4, nesting level increased to 5

(clang)